lib/content-services/src/lib/tag/tags-creator/tags-creator.component.scss

```diff
@@ -27,6 +39,31 @@ adf-tags-creator {
             padding-right: 3px;
             font-size: 14px;
         }
+
+        .mat-form-field-appearance-fill .mat-form-field-flex {
+            background: none;
+        }
+
+        .mat-form-field-appearance-fill .mat-form-field-infix {
+            padding: 0;
+        }
+    }
+
+    .adf-tag-search-field {
+        background: var(--theme-metadata-buttons-background-color);
+        height: 32px;
+        border-radius: 12px;
+        align-items: center;
+        display: flex;
+        padding: 0 12px;
+
+        .mat-form-field-underline {
+            display: none;
+        }
+
+        .mat-form-field-wrapper {
```

Comment:
you should be changing `.mat-form-field`, not internal classes:

```css
 .mat-form-field {
        width: 100%;
    }
```

Reply:
without .mat-form-field-wrapper width, here is the screen record:
[screen-capture (18).webm](https://github.com/Alfresco/alfresco-ng2-components/assets/107834418/24874510-444b-416c-8c25-3ad3573004c7)
without that width, the text is breaking